content/en/users/high-throughput-compute/_index.md

```diff
@@ -97,17 +94,25 @@ The key components of the EGI High Throughput Compute architecture are:
 
 ### Access model
 
-Access to EGI High Throughput Compute resources is based on
-[Virtual Organisations](../check-in/vos), which
-rely on [X.509 proxy certificates with VOMS extensions](../check-in/vos/voms).
-Users have to enroll into one VO before using the service.
-
-VOs are fully managed by research communities, allowing them to manage their
-users and grant access to their services and resources. This means users can
-either own their resources and use EGI services to federate them, or can use
-the resources available in the EGI infrastructure.
-
-Users may gain opportunistic usage to unused resources. These are resources
-that are not dedicated to the users’ organization, but are accessible when the
-research center(s) have some spare resources. This enables the most efficient
-use of resources.
+Access to HTC resources in the EGI infrastructure is based on X.509
+certificates and [Virtual Organisations](../check-in/vos) (VOs).
+
+VOs are fully managed by research communities, allowing communitites to manage
+their users and grant access to their services and resources. This means
+communities can either own their resources and use EGI services to share
+(federate) them, or can use the resources available in the EGI infrastructure
+for their scientific needs.
+
+Before users can access EGI HTC services, they have to:
+
+1. Obtain an X.509 certficate. The certificates are issued by Certification
```

Comment:
Shouldn't this be part of common AAI section?

Reply:
It will be. But this PR tries NOT to touch on the check-in service. I will do that in a separate PR, will add pages for current and legacy auth, and will update this to link there.

Reply:
For now, this information is not present anywhere else in this form.